tasks/section_03_level1.yml

```diff
@@ -9,11 +9,5 @@
   register: sysctl
   with_items:
     - cis-section_03_level1.conf
-  tags: sysctl
-
-- name: ensure apply sysctl configurations
-  systemd:
-    name: systemd-sysctl
-    state: "{{ (sysctl.changed) | ternary('restarted', 'started') }}"
-    enabled: true
-  tags: sysctl
+  notify: "restart sysctl"
```

Comment:
This `notify:` is a alternative to restart and check the `sysctl` status?

Reply:
Yes. 

Take a look : https://docs.ansible.com/ansible/2.5/user_guide/playbooks_intro.html#handlers-running-operations-on-change

Reply:
nice tip! thanks 🤙